### PR TITLE
feat(web-server): add stdout: "pipe"|"ignore" option

### LINKED ISSUE #22454: [Feature] add option to webServer to show logs
Continued from https://discord.com/channels/807756831384403968/1096822303653900368

Something like:

```ts
// ...
	webServer: {
		command: process.env.CI
			? `cross-env PORT=${PORT} npm run start:mocks`
			: `cross-env PORT=${PORT} npm run dev`,
		port: Number(PORT),
		reuseExistingServer: true,
		showLogs: true, // <-- that
	},
// ...
```

Using `DEBUG=pw:webserver` sorta works. For some reason it only logged the startup logs for my server, but didn't log anything after that. In any case, that environment variable is often overloaded for other things as well and it definitely falls short of the DX we've come to expect from Playwright.

### PR DESCRIPTION
**Pull request by mxschmitt**
Fixes https://github.com/microsoft/playwright/issues/22454

### CODE REVIEW

File: docs/src/test-api/class-testconfig.md [hunk 1]
```diff
@@ -599,6 +599,7 @@ export default defineConfig({
   - `ignoreHTTPSErrors` ?<[boolean]> Whether to ignore HTTPS errors when fetching the `url`. Defaults to `false`.
   - `timeout` ?<[int]> How long to wait for the process to start up and be available in milliseconds. Defaults to 60000.
   - `reuseExistingServer` ?<[boolean]> If true, it will re-use an existing server on the `port` or `url` when available. If no server is running on that `port` or `url`, it will run the command to start a new server. If `false`, it will throw if an existing process is listening on the `port` or `url`. This should be commonly set to `!process.env.CI` to allow the local dev server when running tests locally.
+  - `forwardOutput` ?<[boolean]> If `true`, it will forward the stdout of the webserver to the current process. Stderr is always getting forwarded.
```
Comment (dgozman) [inline docs/src/test-api/class-testconfig.md:602]: I'd recommend `stdout: 'ignore' | 'pipe'` or something like that.

**Review by dgozman — COMMENTED** [review]
Let's wait for the branch cut first.

**Review by dgozman — APPROVED** [review]

**Comment by mxschmitt** [comment]
> There seems to have been releases since this was merged, but the option isn't showing up in the TypeScript defs or the docs. When can we expect to get this update?

This got strategically merged after the branch cut, we'll release it as part of v1.34 (if it passes our internal api review).

**Comment by kentcdodds** [comment]
Good to know. Is there a place I can go to learn about the release strategy/cadence for this project?

**Comment by mxschmitt** [comment]
> Good to know. Is there a place I can go to learn about the release strategy/cadence for this project?

Unfortunately not as of today.

**Comment by andrewmclagan** [comment]
Has this been released?

**Comment by mxschmitt** [comment]
> Has this been released?

Yes see here https://playwright.dev/docs/test-webserver#configuring-a-web-server

**Pull request merged**